tests/agent_test.py

```diff
@@ -8,6 +8,9 @@
 
 from agent.tsunami import tsunami
 from agent import tsunami_agent as ts_agt
+from typing import List
+from ostorlab.assets import ipv4 as ipv4_asset
```

Comment:
Please respect the guidlines we use for the imports, standard library /n 3rd party packages /n your code in other files.